request.js

```diff
@@ -71,6 +71,38 @@ function filterOutReservedFunctions (reserved, options) {
   return object
 }
 
+function transformFormData (formData) {
+  // Transform the object representation of form-data fields to array representation.
+  // This might not preserve the order of form fields defined in object representation.
+  // But, this transformation is required to support backward compatibility.
+  //
+  // Form-Data should be stored as an array to respect the fields order.
+  // RFC 7578#section-5.2  Ordered Fields and Duplicated Field Names
+  // https://tools.ietf.org/html/rfc7578#section-5.2
+
+  var transformedFormData = []
+  var appendFormParam = function (key, value) {
+    transformedFormData.push({
+      key: key,
+      value: value && value.hasOwnProperty('value') ? value.value : value,
+      options: value && value.hasOwnProperty('options') ? value.options : undefined
+    })
+  }
+  for (var formKey in formData) {
```

Comment:
Check of `Object.keys().forEach` is faster here.

Reply:
This is discussed earlier: https://github.com/postmanlabs/postman-request/pull/27#discussion_r238590424